src/hotspot/share/opto/runtime.cpp

```diff
@@ -2343,6 +2409,12 @@ void OptoRuntime::initialize_types() {
 #endif // INCLUDE_JVMTI
   _dtrace_method_entry_exit_Type      = make_dtrace_method_entry_exit_Type();
   _dtrace_object_alloc_Type           = make_dtrace_object_alloc_Type();
+  _clone_type_Type                    = make_clone_type_Type();
+#if INCLUDE_SHENANDOAHGC
```

Comment:
A lot of the initializations in this function could be skipped based on runtime flags.  Should we check `UseShenandoahGC` here?